Examples/LambdaFunctions/Sources/LambdaAPI/Product.swift

```diff
@@ -0,0 +1,24 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+
+public struct Product: Codable {
+    public let sku: String
+    public let name: String
+    public let description: String
+    public var createdAt: String?
+    public var updatedAt: String?
```

Comment:
should these be dates?

Reply:
It could be, but I left the String to have a more clear output in the JSON:
With String
```
{
    "updatedAt": "2020-06-20T10:31:39.633Z",
    "description": "Finally! Swift 🚀 + AWS Lambda⚡️️",
    "createdAt": "2020-06-20T10:31:39.633Z",
    "name": "A new template 3",
    "sku": "3"
}
```
With Date
```
{
    "createdAt": 614341899.6329999,
    "description": "Finally! Swift 🚀 + AWS Lambda⚡️️",
    "updatedAt": 614341899.6329999,
    "name": "A new template 3",
    "sku": "3"
}
```

Note that in DynamoDB the date can be saved ad Number o String. In the example is saved as String iso8061.
https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.NamingRulesDataTypes.html#HowItWorks.DataTypes